infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -40,7 +40,7 @@ sites:
     description: "A site to test new releases on"
     releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
     releaseImageName: dpl-cms-source
-    dpl-cms-release: "2024.48.1"
+    dpl-cms-release: "2024.50.0"
     plan: webmaster
     moduletest-dpl-cms-release: "2024.48.1"
```

Comment:
We normally upgrade both webmaster and moduletest environment for these releases.

One might ask why? We do not test anything on the moduletest environment. The [commit](https://github.com/danskernesdigitalebibliotek/dpl-platform/commit/89eee55be90f6bcd788be46ecf67e70d93d160fe) [messages](https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/355) do not provide a clear explanation on why this is - nor for the previous canary site.

In any case if we do not upgrade it now then we need to bump it in a subsequent deployment.

We have done so [for](https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/529/files) [previous](https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/522/files) [releases](https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/535/files).

Reply:
Well, the runbook needs updating then: https://danskernesdigitalebibliotek.github.io/dpl-docs/DPL-Platform/runbooks/how-release-a-new-version-for-approval-testing/

I'll bump it and deploy it (after doublechecking that Philip isn't doing anything)